assets/js/components/spectre/Column.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const Column = (props) => (
+  <div className={classNames(
+    'column',
+    props.classNames
+  )}>
+    {props.children}
+  </div>
+)
```

Comment:
doesn't feel like you get much from this component

Reply:
:woman_shrugging: